model/project_ref.go

```diff
@@ -2032,6 +2032,14 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 		if catcher.HasErrors() {
 			return false, errors.Wrapf(catcher.Resolve(), "validating external links")
 		}
+		var pRef *ProjectRef
+		pRef, err = FindBranchProjectRef(projectId)
+		if err != nil {
+			return false, errors.Wrapf(err, "error getting project")
+		}
+		if p != nil && pRef != nil && p.IsPerfEnabled() != pRef.IsPerfEnabled() && pRef.Identifier != pRef.Id {
```

Comment:
It actually can return nil because (for some reason) internally, we return nil, nil if it is not found. Should this be a separate error case?
I didn't make them error cases because before they weren't but it does make sense since now they are required.